hexa/files/schema/mutations.py

```diff
@@ -38,6 +39,12 @@ def resolve_prepare_download_object(_, info, **kwargs):
         download_url = get_storage().generate_download_url(
             workspace.bucket_name, object_key, force_attachment=True
         )
+        track(
+            request.user,
+            "file_downloaded",
```

Comment:
I'd prefer something that represents the truth such as : `files.generate_download_url`